src/Shopsys/ShopBundle/Resources/styles/front/common/components/list/products.less

```diff
@@ -258,19 +258,25 @@
 
         &__action {
             transition: @transition;
+            text-align: center;
 
             @media @query-vl {
                 display: none;
                 position: absolute;
                 right: 0;
                 bottom: 10px;
                 left: 0;
-                text-align: center;
 
                 .list-products__item:hover & {
                     display: block;
                 }
             }
+
+            .btn--success {
```

Comment:
It's because I don't know how to add class to this:
![image](https://user-images.githubusercontent.com/29253585/44842142-60b51080-ac45-11e8-8441-9f26692bf2f1.png)

Reply:
temporary I did this:
![image](https://user-images.githubusercontent.com/29253585/44842182-7cb8b200-ac45-11e8-990f-0fa47ed93e92.png)

Reply:
oh yeah, here its hard to do it, for demo it doesnt worth it